cmake/hip.cmake

```diff
@@ -1,26 +1,17 @@
-if(DEFINED ENV{HIP_PLATFORM})
-    set(GINKGO_HIP_PLATFORM "$ENV{HIP_PLATFORM}")
-elseif(GINKGO_HIPCONFIG_PATH)
-    execute_process(COMMAND ${GINKGO_HIPCONFIG_PATH}
-        --platform OUTPUT_VARIABLE GINKGO_HIP_PLATFORM)
+cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.21 FATAL_ERROR)
```
3.21?

Reply:
oops, I forgot to adjust the value, thanks!